common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java

```diff
@@ -63,6 +63,9 @@ public class FeatureFlipper {
     @ConfigProperty(name = "notifications.disable-webhook-endpoints-on-failure", defaultValue = "false")
     boolean disableWebhookEndpointsOnFailure;
 
+    @ConfigProperty(name = "ob.backchannel-filler.enabled", defaultValue = "true")
```

Comment:
Is it safe to have this enabled by default?

Reply:
We can have them disabled by default

Reply:
We should enable by default in stage to check that it is working as intended

Reply:
We'll do that with the flag disabled on `master` and enabled for stage in app-interface.